docker/docker-sample-app/README.md

```diff
@@ -1,3 +1,3 @@
 ### Relevant Articles:
 
-- How to Get Docker-Compose to Always Use the Latest Image
+- How to Include Files Outside of Docker’s Build Context
```

Comment:
Let's remove this - we don't update the article list as part of these PRs

Reply:
Quick reminder that this needs reverting

Reply:
Ohh, sorry, it was actually a mistake. I thought I did this change in the directory docker/docker-sample-ecomm-app and the change would go away when I deleted the directory. But it seems I changed thee wrong file when I made this change in the first place.